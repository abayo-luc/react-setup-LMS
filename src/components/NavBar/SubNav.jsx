import React from 'react';
import { Link } from 'react-router-dom';

const SubNav = () => (
  <div id="sub-nav">
    <div className="nav-links">
      <ul className="links">
        {[
          'Economic',
          'Culture',
          'Tech',
          'Politics',
          'Health',
          'Science',
          'Art',
        ].map((text, index) => (
          <li key={`${index + 1}-${text.slice(1, 3)}`}>
            <Link to="/">{text}</Link>
          </li>
        ))}
      </ul>
    </div>
  </div>
);

export default SubNav;
