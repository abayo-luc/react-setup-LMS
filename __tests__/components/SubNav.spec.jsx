import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SubNav from '../../src/components/NavBar/SubNav';

const jSonWarper = toJson(shallow(<SubNav />));

describe('SubNav Component', () => {
  test('should match the snapshot', () => {
    expect(jSonWarper).toMatchSnapshot();
  });
});
