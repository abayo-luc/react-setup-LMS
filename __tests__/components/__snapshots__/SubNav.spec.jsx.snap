// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubNav Component should match the snapshot 1`] = `
<div
  id="sub-nav"
>
  <div
    className="nav-links"
  >
    <ul
      className="links"
    >
      <li
        key="1-co"
      >
        <Link
          to="/"
        >
          Economic
        </Link>
      </li>
      <li
        key="2-ul"
      >
        <Link
          to="/"
        >
          Culture
        </Link>
      </li>
      <li
        key="3-ec"
      >
        <Link
          to="/"
        >
          Tech
        </Link>
      </li>
      <li
        key="4-ol"
      >
        <Link
          to="/"
        >
          Politics
        </Link>
      </li>
      <li
        key="5-ea"
      >
        <Link
          to="/"
        >
          Health
        </Link>
      </li>
      <li
        key="6-ci"
      >
        <Link
          to="/"
        >
          Science
        </Link>
      </li>
      <li
        key="7-rt"
      >
        <Link
          to="/"
        >
          Art
        </Link>
      </li>
    </ul>
  </div>
</div>
`;
